plugins/colorbutton/plugin.js

```diff
@@ -70,21 +84,6 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					attributes: { role: 'listbox', 'aria-label': '' }
 				},
 
-				onRender: function() {
```

Comment:
It is not necessary to add 'onSelectChange' event listener at here. 'onRender' will be fired frequently. So I move the code insided to outside